CONTRIBUTING.md

```diff
@@ -4,20 +4,30 @@ Loki uses GitHub to manage reviews of pull requests:
 
 - If you have a trivial fix or improvement, go ahead and create a pull request.
 - If you plan to do something more involved, discuss your ideas on the relevant GitHub issue.
-
-## Steps to contribute
-
-For now, you need to add your fork as a remote on the original **\$GOPATH**/src/github.com/grafana/loki clone, so:
-
-```bash
-
-$ go get github.com/grafana/loki
-$ cd $GOPATH/src/github.com/grafana/loki # GOPATH is $HOME/go by default.
-
-$ git remote add <FORK_NAME> <FORK_URL>
-```
-
-Notice: `go get` return `package github.com/grafana/loki: no Go files in /go/src/github.com/grafana/loki` is normal.
+- Make sure to follow the prerequisites below before marking your PR as ready for review.
+
+## Pull Request Prerequisites/Checklist
+
+1. Your PR title is in the form `<Feature Area>: Your change`.
+  a. It does not end the title with punctuation. It will be added in the changelog.
+  b. It starts with an imperative verb. Example: Fix the latency between System A and System B.
+  c. It uses Sentence case, not Title Case.
+  d. It uses a complete phrase or sentence. The PR title will appear in a changelog, so help other people understand what your change will be.
+  e. It has a clear description saying what it does and why. Your PR description will be present in the project' commit log, so be gentle to it.
+2. Your PR is well sync'ed with main
+3. Your PR is correctly documenting appropriate changes under the CHANGELOG. You should document your changes there if:
+  * It adds an important feature
+  * It fixes an issue present in a previous release
+  * It causes a change in operation that would be useful for an operator of Loki to know then please add a CHANGELOG entry.
+For documentation changes, build changes, simple fixes etc please skip this step. We are attempting to curate a changelog of the most relevant and important changes to be easier to ingest by end users of Loki.
+4. Your PR documents upgrading steps under `docs/sources/upgrading/_index.md` if it changes:
+  * default configuration values
+  * metric names or label names
+  * changes existing log lines such as the metrics.go query output line
```

Comment:
People outside loki squad usually won't be that familiar with `metrics.go` convention :) 
```suggestion
  * changes existing log lines that may be used in dashboard or alerts. e.g: logs lines in any `metrics.go` files might be getting used in building dashboards or alerts.
```